CHANGES.md

```diff
@@ -1,4 +1,13 @@
 ## Release Notes
+# Changes
+
+## 0.11.0
+* major: changed module name to @axway/swagger-tools
+* major: Removed support for Swagger 1.2
```

Comment:
major: removed swagger-tools binary

There's a few fixes by the look of things too (i notice some regex escaping that i don't quite understand)

Reply:
The regex escaping fixes were regressions after upgrade.